.github/workflows/main-branch.yml

```diff
@@ -19,26 +19,11 @@ jobs:
     name: Python Cross Platform Build & Test
     uses: ./.github/workflows/python-cross-platform-support.yml
 
-# Commenting out from main branch workflow till permissions aspect can be resolved for
-# reconnecting automated page deployment. The below workflow is able to be ran manually
-# allowing for the site to stay up to date.
-  # build_and_deploy_documentation:
-  #   name: Build and Deploy GitHub Pages
-  #   uses: ./.github/workflows/sphinx.yml
-
-  deploy_artifacts:
-    name: Deploy Artifacts
-    runs-on: ubuntu-22.04
-    needs: build_python_artifacts
+  build_and_deploy_documentation:
+    # Set token permissions
     permissions:
-      # Important:  This permission is mandatory for trusted publishing
+      contents: read
+      pages: write
       id-token: write
-    steps:
```

Comment:
Going to ping JD on this again, about us standardizing and all of our other repositories have a dedicated deployment workflow, to please update the PyPi deployment to look for the deploy artifacts flow rather than the main flow for this repository.